test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -90,6 +90,7 @@ public void run() {
                 }
             });
 
+            robot.delay(1000);
             robot.waitForIdle(1000);
 
             Rectangle bounds = frame.getBounds();
```

Comment:
Iguess we missed putting this getBounds and getInsets call in EDT. Maybe that thread context switch can make up for the delay that is introduced now as we already have a 1s wait before screen capture.